simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,24 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    lenLst = len(lst)
+    i = 0
+    listBinary = []
+    carry = 1
+
+    while i < len(lst):
+        if lst[i] + carry == 2:
+            listBinary.append(0)
+            carry = 1
+        else:
+            listBinary.append(lst[i] + carry)
```

Comment:
You are wasting computing power here. You have "lst[i] + carry" calculate two times. Can you temporarily store this one into a "tempSum" variable?

Reply:
need some discuss with this.